src/recaptcha.js

```diff
@@ -152,4 +155,5 @@ ReCAPTCHA.defaultProps = {
   tabindex: 0,
   size: "normal",
   badge: "bottomright",
+  hl: "en",
```

Comment:
💡 i think we should "default" it to `undefined` so it is backward compatible?

Reply:
wondering, if we default it to undefined,  in that case wouldn't the user must define  on every case ?

Reply:
https://developers.google.com/recaptcha/docs/display#js_param

Reply:
It does say that when you do not supply a language, `hl`, it attempts to derive the users language.
> Auto-detects the user's language if unspecified.

Also, i assumed the `js param`'s were only for when loading the script like the way `react-google-recaptcha` currently tries to use `hl`, but again this is awesome if this works!

Reply:
If this work we can remove the hl parameter from the URL I guess and pass it on the JS side only. and so it would go props || globalOption || en.

Reply:
Great, I tried a codesandbox example using my fork. please, check!!
https://codesandbox.io/s/blissful-haze-ozk9y

Reply:
I don't think we should "default" to `'en'`. We should let the google decide the default language, `Auto-detects the user's language if unspecified.`

Reply:
@hartzis ah good point, so yeah props || globalOption || undefined.

Reply:
@hartzis  makes sense. lets not default.